packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -204,6 +230,9 @@ class MenuWithFlyout extends React.Component {
           to="#default-link2"
           itemId={3}
           isActive={activeItem === 3}
+          isExpandable
+          onMouseEnter={handleMenuToggle}
+          flyoutMenuItems={this.state.isSubMenuOpen ? flyoutMenuItems : ""}
```

Comment:
at the moment the menu also takes the full width of the container rather than how it is in the core example when the `pf-m-flyout` modifier is applied. 

is this to be expected? see https://www.patternfly.org/v4/components/menu#with-flyout